ocs_ci/ocs/resources/objectbucket.py

```diff
@@ -108,126 +199,172 @@ def internal_verify_health(self):
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def internal_verify_deletion(self):
+        """
+        Abstract deletion verification method
+
+        """
+        raise NotImplementedError()
+
 
-class S3Bucket(MCGBucket):
+class MCG_CLIBucket(ObjectBucket):
```

Comment:
change to MCGCLIBucket or MCGCliBucket to comply with https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html

Reply:
I know the best practice is to not use underscores - but I felt like MCGCLIBucket is too confusing, and renaming it to McgCliBucket won't be much better since it won't be obvious those are acronyms.
Do you think there might be a better way to solve this issue?